test/acceptance/features/bindAppToService.feature

```diff
@@ -756,3 +756,36 @@ Feature: Bind an application to a service
             """
         Then Service Binding "binding-request-cross-ns-service" is ready
         And The application env var "BACKEND_HOST_CROSS_NS_SERVICE" has value "cross.ns.service.stable.example.com"
+
+    @negative
+    Scenario: Service binding with a missing app and configMap as a service
+        Given ConfigMap is applied
+            """
+            apiVersion: v1
+            kind: ConfigMap
+            metadata:
+                name: example
+                annotations:
+                    service.binding: path={.data},elementType=map
+            data:
+                word: "hello"
+            """
+        When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-configmap
+            spec:
+                services:
+                -   group: ""
+                    version: v1
+                    kind: ConfigMap
+                    name: example
+            """
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-configmap" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-configmap" should be changed to "False"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding "binding-request-configmap" should be changed to "EmptyApplication"
+        And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "binding-request-configmap" should be changed to "True"
+        And Secret "binding-request-configmap" contains "CONFIGMAP_DATA_WORD" key with value "hello"
```

Comment:
is there are a way to inject bindings using the original keys from configmap/secret. For example, to inject env var `WORD` with value `hello`?

Reply:
This has been implemented in https://github.com/redhat-developer/service-binding-operator/pull/407 and https://github.com/redhat-developer/service-binding-operator/pull/522.

Reply:
@isutton what should be the syntax to achieve this?

Reply:
I was referring to including the kind in the name for disambiguation purposes. It has appeared in both PRs and was pointing out so you can check what has been implemented.